quickstart/pom.xml

```diff
@@ -145,6 +153,20 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <source>1.8</source>
+              <target>1.8</target>
+              <annotationProcessorPaths>
```

Comment:
Don't we need to include javapoet here? As said elsewhere, it's probably safer to explicitly declare our annotation processor as described [here](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#annotationProcessors).

Reply:
Everything works now without it, I had problems probably because I missed doing `mvn install` on the extension itself.